### PR TITLE
Revenants have random names

### PR DESCRIPTION
**Pull request by XDTM**
:cl: XDTM
add: Revenants now have randomly generated names.
/:cl:

The name structure is "[spirit_type] of [adverb] [theme]", picked from a suitably edgy list.

### CODE REVIEW

File: strings/revenant_names.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"suffering",
+	"death",
+	"destruction",
+	"hate",
+	"misery",
+	"disease",
+	"desperation",
+	"sorrow",
+	"ruin"
+    ]
```
Comment (DaxDupont) [inline strings/revenant_names.json]: Add salt

**Comment by DaxDupont** [comment]
♥

**Comment by KingPhilipIII** [comment]
@XDTM 

Can we please have a small chance for Griff Ghost?

**Comment by Dorsisdwarf** [comment]
add "clowns" as a theme

**Comment by MoreRobustThanYou** [comment]
soul of hateful salt

File: strings/revenant_names.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (MoreRobustThanYou) [inline strings/revenant_names.json]: run this through [jsonlint ](https://jsonlint.com/)please

File: strings/revenant_names.json (hunk 1 above)
Comment (Supermichael777) [reply strings/revenant_names.json]: revenents are annihilated by salt which makes their ability to attack players weird

**Comment by Polyphynx** [comment]
Aww, it's just the _Spirit of peaceful **hellfire**_

**Comment by davethwave** [comment]
Would be interesting if you could add names of the people who died traumatically to the naming roster. For instance being gibbed in any way, lings lethal DNA extraction method, burned to death, or thrown into space.

**Comment by Armhulen** [comment]
DUDE

DYING SETS THE COLOR OF YOUR REVENANT

WE /SIMS/ NOW BOIZ

**Comment by XDTM** [comment]
I was going to code several revenant types but it turns out that's a lot of work

**Comment by Armhulen** [comment]
hey that was actually something i was going to do

**Comment by Armhulen** [comment]
there were three classes of ghosts a thin green one called the polstergeist that threw objects, revenant as it is and a possesser one that fully took over bodies and any damage the body would take would damage the possesser

never came into fruition

**Comment by XDTM** [comment]
Or going with the based-on-the-type-of-death theme we could have:
Brute damage: violent red ghost with very physical interactions (throwing, shaking, maybe explosions?)
Burn damage: orange ghost with abilities that affect the atmosphere, affecting gas and temperature. Maybe stuff like freezing touch, fireburst, so on.
Toxin damage: subtle green ghost, would have abilities that impair or hurt without being noticed; covertly injecting some toxins, a contagion maybe.
Oxygen damage: A transparent blue ghost, it would cause unconsciousness/sleep at the worst times, then suck the oxygen out of the air.
Brain damage: A crazed pink or grey ghost, would cause hallucinations with precise control over them, and maybe scale up to possession.
Clone damage: A malformed ghost, it would attack by spawning flesh horrors of various kinds.

**Comment by fludd12** [comment]
Names remind me of abysmal exalted. :+1:

**Comment by Buggy123** [comment]
>Can we please have a small chance for Griff Ghost?

**Comment by ShadowDeath6** [comment]
there's a chance for ghost of brutal grief which seems close enough

**Comment by XDTM** [comment]
By the way, this was the inspiration for the PR
https://www.youtube.com/watch?v=jAAy8yIgseU

File: strings/revenant_names.json (hunk 2 above)
Comment (duncathan) [reply strings/revenant_names.json]: yes please lint this

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by vuonojenmustaturska** [comment]
We have json verification as a part of travis, also the file checks out fine

**Pull request merged**